session_4/blogspot/app/views/articles/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <h1 class="display-2"><strong><%= @article.title %></strong></h1>
+
+      <p class="h3 text-muted">
+        <%= @article.topic %>
+      </p>
+
+       <p class="h6">
+       <br>
+        <%= @article.content %>
+      </p>
+    </div>
+    <div class="col">
+      <br>
+      <%if @article.image.attached?%>
+        <%= image_tag @article.image, style: widht="400", class: 'img-fluid rounded float-left'%>
+      <%end%>
+    </div>
+  </div>
+<%#even we can check the condition here too%>
+  <%if @goodboy%>
+    <%= link_to 'Edit', edit_article_path(@article), remote: false, class: 'btn text-white btn-primary'%>
```

Comment:
Use different styles for different buttons

Reply:
Sure